apps/examples/src/misc/Bisect/pr-numbers.ts

```diff
@@ -0,0 +1,5 @@
+// This file is automatically generated by scripts/get-pr-numbers.ts.
+// Do not edit manually. Or do, I'm a comment, not a cop.
+
+/** @public */
+export const prNumbers = []
```

Comment:
Committing this with an empty array as it will be generated while running bisect. We probably need to have this one committed in, otherwise you might get compile errors until you run bisect for the first time.